features/page_objects/google_search.rb

```diff
@@ -0,0 +1,32 @@
+require 'page-object'
+
+class GoogleSearch
+  include PageObject
+
+  URL = 'https://www.google.com'
+
+  text_field(:search_box, name: 'q')
+  button(:search_button, name: 'btnK')
+  div(:search_results, id: 'resultStats')
+
+  def visit_google_search
+    @browser.goto URL
+  end
+
+  def enter_search_keyword(keyword)
```

Comment:
you don't need this, page-object creates these methods for you. to use it you do this:
on_page(GoogleSearch).search_box = 'nextfilx'